src/main/java/fact/statistics/TimeseriesFeatures.java

```diff
@@ -32,8 +32,9 @@
 	private int searchWindowRight = 250;
 	@Parameter(required=true,description="name of the key of the calculated features")
 	private String outputKey = null;
-	
-	
+	@Parameter(required=false, description="Whether to do the calculation with substraction of the avarageData.")
```

Comment:
Maybe explain, what this will mean for the features. 
Use longer name: e.g. `subtractSmoothedData`